game/src/contrib/components/StatsComponent.java

```diff
@@ -1,38 +1,57 @@
 package contrib.components;
 
+import contrib.utils.components.health.DamageType;
 import contrib.utils.components.stats.DamageModifier;
 
 import core.Component;
 import core.Entity;
 
-public class StatsComponent extends Component {
+/**
+ * Add resistances and vulnerabilities for {@link contrib.utils.components.health.DamageType}s to
+ * the associated entity.
+ *
+ * <p>To set resistances and vulnerabilities for a damage type, use {@link #multiplier(DamageType,
+ * float)}. For resistances, use a float less than 1, for vulnerabilities use a float greater than
+ * 1.
+ *
+ * <p>The modifier values are taken into account by the {@link contrib.systems.HealthSystem} when
+ * calculating damage, allowing for defining resistances and vulnerabilities for different damage
+ * types.
+ *
+ * @see contrib.utils.components.stats.DamageModifier
+ */
+public final class StatsComponent extends Component {
 
-    private DamageModifier damageModifier = new DamageModifier();
+    private final DamageModifier damageModifier = new DamageModifier();
 
     /**
-     * Create a new component and add it to the associated entity
+     * Create a new component and add it to the associated entity.
      *
      * @param entity associated entity
      */
-    public StatsComponent(Entity entity) {
+    public StatsComponent(final Entity entity) {
         super(entity);
     }
 
     /**
-     * Get the DamageModifier object of the entity
+     * Get multiplier for a given damage type
```

Comment:
Was passiert, wenn ich einen Multiplikator abfrage, für den ich nix habe? Kann bzw. darf das passieren? Und falls ja: Was kommt dann zurück?

Ich glaube, da wird eine `1.0` geliefert. Aber das sollte kurz erklärt werden.

Reply:
hattest recht, habs in die doc geschrieben